pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: reflectable
-version: 2.2.1
+version: 2.2.1+2
 description: >
   Reflection support based on code generation, using 'capabilities' to
   specify which operations to support, on which objects.
 homepage: https://www.github.com/dart-lang/reflectable
 environment:
   sdk: '>=2.3.0 <3.0.0'
 dependencies:
-  analyzer: '>=0.39.0 <0.40.0'
+  analyzer: '>=0.39.4 <0.40.0'
```

Comment:
Because we use internals of the analyzer I think we should make the upper bound tight on the highest known working version.